internal/sdkprovider/service/organization/organization_user.go

```diff
@@ -66,100 +72,55 @@ eliminate the member from the organization if one has accepted an invitation pre
+Invitation of organization users is not supported anymore via Terraform. Therefore 
+creation of this resource is not supported anymore. We reccoemnd to use WebUI to create
+and organization user invitation. And upon receiving an invitation, a user can accept it 
+using WebUI. Once accepted, the user will become a member of the organization and will 
+be able to access it via Terraform.
+		`,
 	}
 }
 
-func resourceOrganizationUserCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	client := m.(*aiven.Client)
-	organizationID := d.Get("organization_id").(string)
-	userEmail := d.Get("user_email").(string)
-
-	err := client.OrganizationUserInvitations.Invite(ctx, organizationID, aiven.OrganizationUserInvitationAddRequest{
-		UserEmail: userEmail,
-	})
-	if err != nil {
-		return diag.FromErr(err)
-	}
-
-	d.SetId(schemautil.BuildResourceID(organizationID, userEmail))
-
-	return resourceOrganizationUserRead(ctx, d, m)
+// resourceOrganizationUserCreate create is not supported anymore
+func resourceOrganizationUserCreate(_ context.Context, _ *schema.ResourceData, _ interface{}) diag.Diagnostics {
+	return diag.Errorf("creation of organization user is not supported anymore via Terraform. " +
+		"Please use WebUI to create an organization user invitation. And upon receiving an invitation, " +
+		"a user can accept it using WebUI. Once accepted, the user will become a member of the organization " +
+		"and will be able to access it via Terraform using datasource `aiven_organization_user`")
 }
 
+// resourceOrganizationUserRead reads the properties of an Aiven Organization User and provides them to Terraform
 func resourceOrganizationUserRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	var found bool
 	client := m.(*aiven.Client)
 
 	organizationID, userEmail, err := schemautil.SplitResourceID2(d.Id())
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	r, err := client.OrganizationUserInvitations.List(ctx, organizationID)
+	rm, err := client.OrganizationUser.List(ctx, organizationID)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	for _, invite := range r.Invitations {
-		if invite.UserEmail == userEmail {
-			found = true
+	for _, user := range rm.Users {
+		userInfo := user.UserInfo
 
+		if userInfo.UserEmail == userEmail {
 			if err := d.Set("organization_id", organizationID); err != nil {
 				return diag.FromErr(err)
 			}
-			if err := d.Set("user_email", invite.UserEmail); err != nil {
+			if err := d.Set("user_email", userInfo.UserEmail); err != nil {
 				return diag.FromErr(err)
 			}
-			if err := d.Set("invited_by", invite.InvitedBy); err != nil {
+			if err := d.Set("create_time", user.JoinTime.String()); err != nil {
 				return diag.FromErr(err)
 			}
-			if err := d.Set("create_time", invite.CreateTime.String()); err != nil {
+			if err := d.Set("user_id", user.UserID); err != nil {
 				return diag.FromErr(err)
```

Comment:
Do you expect create_time and invited_by to disappear?

Reply:
`create_time` will stay and `invited_by` is getting deprecated, we no longer can get data for this field since we no longer can invite people or read the invitations.